src/actor/mod.rs

```diff
@@ -105,15 +105,33 @@ impl Actor {
         &self,
         operation: impl AsRef<str>,
         payload: Option<impl Into<Vec<u8>> + AsRef<[u8]>>,
+    ) -> anyhow::Result<Result<Option<Vec<u8>>, String>> {
+        self.call_with_context(operation, payload, None::<Vec<u8>>)
+            .await
```

Comment:
Use `call` directly

Reply:
I thought here it would be better to reduce code duplication by having this function call `call_with_context` internally

Reply:
Revisiting this one, actually